src/lib.rs

```diff
@@ -41,6 +41,34 @@ pub mod ctypes {
     pub use core::ffi::c_void;
 }
 
+// Confirm that our type definitions above match the actual type definitions.
+#[cfg(test)]
+static_assertions::assert_eq_size!(ctypes::c_char, libc::c_char);
+#[cfg(test)]
```

Comment:
Perhaps a little cleaner to nest all this stuff like:

```
#[cfg(test)]
mod assertions {
    static_assertions::assert_type_eq_all!(ctypes::c_schar, libc::c_schar);
    static_assertions::assert_type_eq_all!(ctypes::c_uchar, libc::c_uchar);
    ...
}
```
?

Reply:
Good idea; done.